CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 2.3.0-beta.1
+
+### ✨ Features and improvements
+* Add configurable fog as a root style specification ([#10564](https://github.com/mapbox/mapbox-gl-js/pull/10564))
+* Add `map.queryTerrainElevation(lngLat)` API ([#10602](https://github.com/mapbox/mapbox-gl-js/pull/10602))
```

Comment:
```suggestion
* Add client-side elevation querying with `map.queryTerrainElevation(lngLat)` when terrain is active ([#10602](https://github.com/mapbox/mapbox-gl-js/pull/10602))
```

Reply:
Feel free to rephrase this one, but I find it important to highlight two things:
- Available when terrain is on
- Prevents you from doing any sort of server query